docs/winrates.md

```diff
@@ -0,0 +1,22 @@
+## Winrates
+We periodically (especially after large changes) conduct a large amount of games to measure the winrates with our best strategies
+on all the characters. 
+
+### Current Stats:
+From October 2024: [release-03](https://github.com/xaved88/bottled_ai/releases/tag/release-03)
+
```

Comment:
Think we should link to the detailed winrates document you created

Reply:
Ah agreed, I thought I had - good catch!